payments/swish/redirect.md

```diff
@@ -181,7 +181,7 @@ initiate the sales transaction.
 ## API Requests
 
 The API requests are displayed in the [purchase flow](#purchase-flow-3). Swish
-is a one-phase payment instrument that is based on sales transactions not involving
+is a one-phase payment method that is based on sales transactions not involving
```

Comment:
Didn't we agree to use "instrument" in the documentation?

```suggestion
is a one-phase payment instrument that is based on sales transactions not involving
```

Reply:
We sure did! My bad!